### ISSUE TITLE
Is there an example of successfully finding a bug or error?

### ISSUE THREAD

**Issue by st0mb1e** [marmelab/gremlins.js#199]
Is there an example of successfully finding a bug or error?
If so, can you attach an example of an open source project that was fuzzed?

Thank you in advance.

**Comment by Piedone** [comment]
I won't be able to provide a specific example from the top of my head, but in our https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions solution, gremlins.js found bugs many times before. In our other applications too.

**Comment by axolotl-logic** [comment]
> I won't be able to provide a specific example from the top of my head, but in our https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions solution, gremlins.js found bugs many times before. In our other applications too.

Do you still use it? It appears abandoned?

**Comment by Piedone** [comment]
Yes, we still use it. Since it works and there are no bugs that would bother us, we just keep on using it.
